en/features/census_configuration.md

```diff
@@ -50,43 +53,43 @@ The information to be filled in is divided into three sections:
 
   Required fields for the request:
 
-  - **Request method name**: Request method name accepted by the City Census WebService.
+    - **Request method name**: Request method name accepted by the City Census WebService.
```

Comment:
At least on the preview on GitHub, the layout seems broken in this part after the indentation changes :thinking:.

Reply:
Ok, I'll check it out, although in the preview in my text editor it apparently looked fine.
![Captura de pantalla 2023-04-19 a las 9 36 31](https://user-images.githubusercontent.com/16189/233003166-e1166851-67d0-43c7-9213-1703ee0abae7.png)